app/styles/vars.scss

```diff
@@ -1,3 +1,3 @@
-$page-top-margin: 20px;
+$page-top-margin: 14px;
```

Comment:
@CosmicCoder96 this will not affect other areas as now when we go to mobile view the height of nav-bar changes (decreases) so decreasing `$page-top-margin` which is used in `.public-event` and `.main-container` will cover the space created due to reduced nav-bar.